src/index.js

```diff
@@ -12,8 +12,12 @@ import Home from './js/home'
 import AuthPage from './js/authPage'
 import Question from './js/question'
 
-const NotFound = props => <h1>Sorry, could not find the page with URL <code>{props.url}</code></h1>
-
+const NotFound = props => <div>
```

Comment:
Yes and instead import it in index and export default from the module

Regards,
Megh Parikh
On 01-Jul-2016 19:07, "Anubhab Sen" <notifications@github.com> wrote:

> In src/index.js
> <https://github.com/felicity-buzz-2k16/quiz-portal-frontend/pull/4#discussion_r69300324>
> :
>
> > @@ -12,8 +12,12 @@ import Home from './js/home'
> >  import AuthPage from './js/authPage'
> >  import Question from './js/question'
> >
> > -const NotFound = props => <h1>Sorry, could not find the page with URL <code>{props.url}</code></h1>
> > -
> > +const NotFound = props => <div>
>
> So I remove the not found thingy entirely from index?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/felicity-buzz-2k16/quiz-portal-frontend/pull/4/files/df8bcebac88767a7479cdcc84d0a50769e616194#r69300324>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFs91XrsxBMWIl9fScttreOblCgjbwNqks5qRRgygaJpZM4JDIFp>
> .
>